tests/unit/executors/indexers/test_numpyindexer.py

```diff
@@ -19,17 +18,17 @@
 cur_dir = os.path.dirname(os.path.abspath(__file__))
 
 
-@pytest.mark.parametrize('batch_size, compress_level', [(None, 0), (None, 1), (2, 0), (2, 1)])
-def test_numpy_indexer(batch_size, compress_level):
+@pytest.mark.parametrize('batch_size, compress_level', [(None, 0)])
+def test_numpy_indexer(batch_size, compress_level, tmpdir):
+    index_dir = tmpdir.join('NumpyIndexer-test.bin')
     with NumpyIndexer(index_filename='np.test.gz', compress_level=compress_level) as indexer:
         indexer.batch_size = batch_size
         indexer.add(vec_idx, vec)
-        indexer.save()
+        indexer.save(index_dir)
```

Comment:
So u do not need to set this up because later may not be able to read from dump automatically

Reply:
check, I created a fixture for meta

Reply:
Love it!